src/components/ArtObjectPageComponents/PanelDetails/index.jsx

```diff
@@ -247,9 +277,28 @@ class PanelDetails extends Component {
       imageLoaded: false,
       activeImageIndex: 0,
       thumbnailsOpen: false,
+      renditions: null,
     };
   }
 
+
+  /** Async method to fetch possible additional renditions for this object */
+  async componentDidUpdate() {
+    if (this.props.object.id && this.state.renditions === null) {
+      try {
+        const response = await axios({
+          method: 'GET',
+          url: `/api/objects/${this.props.object.id}/assets`
+        });
+  
+        this.setState({...this.state, renditions: response.data || []})
+      } catch (error) {
+        console.error(`An error occurred fetching renditions`, error);
+      }
+    }
+  }
```

Comment:
We reach out to the API only when the component has updated and we have the Object ID available, and only once, so we're not reaching out for renditions multiple times.